src/lwc/boutiqueAppointments/boutiqueAppointments.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Created by Mariya on 27.01.2023.
+ */
+
+import {LightningElement, api, wire, track} from 'lwc';
+import getAppointmentResourcesByBoutiqueId from '@salesforce/apex/CaseController.getAppointmentResourcesByBoutiqueId';
+
+const COLS = [
+    {
+        label: 'Boutique', fieldName: 'Boutique', type: 'url',
+        typeAttributes: {label: {fieldName: 'boutiqueURL'}}
+    },
+    {
+        label: 'Customer', fieldName: 'Customer', type: 'url',
+        typeAttributes: {label: {fieldName: 'customerURL'}}
+    },
+    {label: 'Phone', fieldName: 'Phone', type: 'phone'},
+    {label: 'Email', fieldName: 'Email', type: 'email'},
+    {label: 'Purpose of Visit', fieldName: 'Visit'},
+    {label: 'Date', fieldName: 'AppointmentDate'},
+    {
+        label: 'Start time', fieldName: 'StartTime', type: 'date', typeAttributes: {
+            hour: '2-digit',
+            minute: '2-digit',
+            second: '2-digit',
+            hour12: true
+        }
+    },
+    {
+        label: 'Comments', type: 'customComment',
+        typeAttributes: {
+            comment: {fieldName: 'Comments'}
+        }
+    }
+];
+
+export default class BoutiqueAppointments extends LightningElement {
+    @api recordId;
+    @track data = [];
+    @track columns = COLS;
+
+    @wire(getAppointmentResourcesByBoutiqueId, {boutiqueId: '$recordId'})
+    wiredCases({error, data}) {
+        this.data = data;
+        if (this.data) {
```

Comment:
MAJOR
Handle error scenario and display toast with error message in case of any error.